app/addons/documents/assets/less/revisionTree.less

```diff
@@ -0,0 +1,15 @@
+.visualizeRevTree {position: relative}
+.visualizeRevTree * {margin: 0; padding: 0; font-size: 10px}
+.visualizeRevTree line{stroke: #000; stroke-width: .50}
+.visualizeRevTree div{position: relative; }
+.visualizeRevTree circle{stroke: #000; stroke-width: .10}
+.visualizeRevTree circle.leaf{fill: #008000;}
+.visualizeRevTree circle.winner{fill: #FF0000}
+.visualizeRevTree circle.deleted{fill: #808080}
+.visualizeRevTree circle{transition: .3s}
+.visualizeRevTree circle.selected{stroke-width: .3}
+.visualizeRevTree div.box{background: #ddd; border: 1px solid #bbb; border-radius: 7px; padding: 7px; position: absolute;}
+.visualizeRevTree .editor {width: 220px}
+.visualizeRevTree .editor dt{width: 100px; height: 15px; float: left;}
+.visualizeRevTree .editor dd{width: 100px; height: 15px; float: left;}
+.visualizeRevTree .editor input{width: 100%; height: 100%}
```

Comment:
Has the CSS been modified at all, or is it all from that external lib? Put any external libs into a `/external` subfolder of assets so it's clear what's ours and what's external.